### PR TITLE
finished adding routes

### PR DESCRIPTION
**Pull request by Rogerch99**
added all the routes

### CODE REVIEW

**Comment by Rogerch99** [comment]
finished adding the routes

File: study_hour/server/index.js [hunk 1]
```diff
@@ -42,56 +42,68 @@ app.get('/Login', (req, res) => {
 // Get all the comments for a location
 // Returned comment should have all comment data including user_id
 app.post('/api/Location/Comments', function (req, res, next) {
-
+    
+    console.log("incomming request", req.body.location_id);
```
Comment (ElliotVilhelm) [inline study_hour/server/index.js]: remove

File: study_hour/server/index.js [hunk 2]
```diff
@@ -42,56 +42,68 @@ app.get('/Login', (req, res) => {
...
+    pgClient.query('SELECT * FROM Comments t WHERE t.location_id= $1 ', [req.body.location], function (err, result) {
+        if (err) {
+            return next(err)
+        }
+        console.log("db result: ", result.rows);
```
Comment (ElliotVilhelm) [inline study_hour/server/index.js]: remove

File: study_hour/server/index.js [hunk 3]
```diff
@@ -42,56 +42,68 @@ app.get('/Login', (req, res) => {
...
+        res.send({dbresponse: result.rows})
+    });
 
 });
 
 // Get all comments from a given user
 // not that high priority
 app.post('/api/User/Comments', function (req, res, next) {
+    console.log("incomming request", req.body.location);
```
Comment (ElliotVilhelm) [inline study_hour/server/index.js]: remove

File: study_hour/server/index.js [hunk 4]
```diff
@@ -42,56 +42,68 @@ app.get('/Login', (req, res) => {
...
+    pgClient.query('SELECT * FROM Comments t WHERE t.user_id= $1 ', [req.body.location], function (err, result) {
+        if (err) {
+            return next(err)
+        }
+        console.log("db result: ", result.rows);
```
Comment (ElliotVilhelm) [inline study_hour/server/index.js]: remove

File: study_hour/server/index.js [hunk 5]
```diff
@@ -42,56 +42,68 @@ app.get('/Login', (req, res) => {
...
+        res.send({dbresponse: result.rows})
+    });
 
 
 });
 
 // Get all data about a user
 app.post('/api/User', function (req, res, next) {
 
+    console.log("incomming request", req.body.user_id);
```
Comment (ElliotVilhelm) [inline study_hour/server/index.js]: same

File: study_hour/server/index.js [hunk 6]
```diff
@@ -42,56 +42,68 @@ app.get('/Login', (req, res) => {
...
+    pgClient.query('SELECT * FROM Users t WHERE t.user_id= $1 ', [req.body.user_id], function (err, result) {
+        if (err) {
+            return next(err)
+        }
+        console.log("db result: ", result.rows);
```
Comment (ElliotVilhelm) [inline study_hour/server/index.js]: remove

File: study_hour/server/index.js [hunk 7]
```diff
@@ -42,56 +42,68 @@ app.get('/Login', (req, res) => {
...
+        
         const config = {
             secret: "supersecret"
         };
         const token = jwt.sign({ id: result.rows[0].user_id }, config.secret, {
             expiresIn: 86400 // expires in 24 hours
         });
-        res.send({auth: result.rows[0].case, token: token})
+        res.send({auth: true, token: token})
+
```
Comment (ElliotVilhelm) [inline study_hour/server/index.js]: delete newline

**Comment by ElliotVilhelm** [comment]
🍏

**Pull request merged**